Jenkinsfile.conda

```diff
@@ -7,6 +19,7 @@ pipeline {
     parameters {
         string(defaultValue: '5.1.0', description: 'The XML Version, exclude any preceeding "v" characters: X.Y.Z', name: 'xml_version')
         string(defaultValue: '4.1.2', description: 'The ts_sal version, exclude any preceeding "v" characters: X.Y.Z', name: 'sal_version')
+        choice choices: ['Bleed', 'Daily', 'Release'], description: 'The upstream build type (Bleed, Daily or Release). This determines from where to pull the RPM.', name: 'build_type'
```

Comment:
Should default be "Release"?

Reply:
Switched the order to ['Release', 'Daily', 'Bleed'], which makes Release (the first option) the default.